src/environment.ts

```diff
@@ -11,26 +11,51 @@ export enum PlatformEnvironment {
 export const DEFAULT_ENVIRONMENT = PlatformEnvironment.Prod as const;
 export const DEFAULT_REGION = Region.US as const;
 export const DEFAULT_RETRY_AFTER = 5000;
-export const DEFAULT_MAX_RETRIES = 50;
+export const DEFAULT_MAX_RETRIES = 10;
 export const DEFAULT_TIME_MULTIPLE = 2;
 
-export type PlatformUrlOptions = {
+export type PlatformOptions = {
```

Comment:
@y-lakhdar brought up a good point that renaming this is potentially a breaking change. Even though I'm not sure if there's much of a reason for implementers to actually import this type, it's still publicly exported.

That being said, does this still fall under "minor release" so long as we clarify that this is a breaking change?

Reply:
breaking is breaking. But you can:
```suggestion
@deprecated use @link {PlatformOptions}
export type PlatformUrlOptions = PlatformOptions;
export type PlatformOptions = {
```

Reply:
Great suggestion! Applied ✅